tikv-control: make a parameter in right places (#11026)
pingcap/docs-cn: tikv-control.md

    -V, --version                 Prints version information

OPTIONS:
        --ca-path <ca-path>              Set the CA certificate path
        --cert-path <cert-path>          Set the certificate path

>         --config <config>                TiKV config path, by default it's <deploy-dir>/conf/tikv.toml

Reviewer: ```suggestion
        --config <config>                TiKV config path, by default it is <deploy-dir>/conf/tikv.toml
```

    store:"127.0.0.1:20160" compact db:KV cf:default range:([], []) success!
    ```

- **本地模式**：

    - 通过 `--data-dir` 选项来指定本地 TiKV 数据的目录路径。
    - 通过 `--config` 选项来指定本地 TiKV 配置文件到路径。

>     在此模式下，需要停止正在运行的 TiKV 实例。

Author: ```suggestion
  在此模式下，需要停止正在运行的 TiKV 实例。
```

### 手动 compact 单个 TiKV 的数据

`compact` 命令可以对单个 TiKV 进行手动 compact。

- `--from` 和 `--to` 选项以 escaped raw key 形式指定 compact 的范围。如果没有设置，表示 compact 整个 TiKV。
- `--region` 选项指定 compact Region 的范围。如果设置，则 `--from` 和 `--to` 选项会被忽略。

> - `--db` 选项可以指定要 compact 的 RocksDB，可选值为`kv` 和 `raft`。

Reviewer: ```suggestion
- `--db` 选项可以指定要 compact 的 RocksDB，可选值为 `kv` 和 `raft`。
```

- `--threads` 选项可以指定 compact 的并发数，默认值是 8。一般来说，并发数越大，compact 的速度越快，但是也会对服务造成影响，所以需要根据情况选择合适的并发数。
- `--bottommost` 选项可以指定 compact 是否包括最下层的文件。可选值为 `default`、`skip` 和 `force`，默认为 `default`。
    - `default` 表示只有开启了 Compaction Filter 时 compact 才会包括最下层文件。
    - `skip` 表示 compact 不包括最下层文件。
    - `force` 表示 compact 总是包括最下层文件。

- 在本地模式 compact data，执行如下命令：

    ```shell
    tikv-ctl --data-dir /path/to/tikv compact --db kv

>     success!

Reviewer: ```suggestion
```

    ```

- 在远程模式 compact data，执行如下命令：

    ```shell

>     $ tikv-ctl --host ip:port compact --db kv

Reviewer: ```suggestion
    tikv-ctl --host ip:port compact --db kv
```

>     success!

Reviewer: ```suggestion
```

### 手动 compact 单个 TiKV 的数据

`compact` 命令可以对单个 TiKV 进行手动 compact。

- `--from` 和 `--to` 选项以 escaped raw key 形式指定 compact 的范围。如果没有设置，表示 compact 整个 TiKV。

> - `region` 选项指定 compact Region 的范围。如果设置，则 `--from` 和 `--to` 选项会被忽略。

Reviewer 2: ```suggestion
- `--region` 选项指定 compact Region 的范围。如果设置，则 `--from` 和 `--to` 选项会被忽略。
```

> - `--host` 选项可以指定要 compact 的 TiKV。

Reviewer 2: 删除，这是 tikv-ctl 通用参数，不是该命令的参数

- `--db` 选项可以指定要 compact 的 RocksDB，可选值为`kv` 和 `raft`。

> - `--data-dir` 选项指定本地 TiKV 数据的目录路径。

Reviewer 2: 同上
